src/api/meeting/index.ts

```diff
@@ -269,3 +259,31 @@ export const updateMeeting = async (meetingId: string, formData: FormType) => {
 
   return response;
 };
+
+interface GetPresignedUrlResponse {
+  url: string;
+  fields: {
+    'Content-Type': string;
+    key: string;
+    bucket: string;
+    'X-Amz-Algorithm': string;
+    'X-Amz-Credential': string;
+    'X-Amz-Date': string;
+    Policy: string;
+    'X-Amz-Signature': string;
+  };
+}
+export const getPresignedUrl = async (contentType: string) => {
+  const { data } = await api.get<Data<GetPresignedUrlResponse>>('/meeting/v1/presigned-url', {
+    params: { contentType },
+  });
+  return data;
+};
+export const uploadImage = async (file: File, url: string, fields: { [key: string]: string }) => {
+  const formData = new FormData();
+  Object.entries(fields).map(([key, value]) => {
+    formData.append(key, value);
+  });
```

Comment:
```suggestion
  for (const [key, value] of Object.entries(fields)) {
    formData.append(key, value);
  }
```
이렇게 하면 map으로 새 배열 생성하지 않아도 된다는 점에서 좋을 것 같아요~!

Reply:
오오 좋은 것 같아! 요렇게 한번 해볼게~~

Reply:
그러면 forEach로만 바꿔도 될듯 !

Reply:
[27249b7](https://github.com/sopt-makers/sopt-crew-frontend/pull/289/commits/27249b79d3d436f7e73407440aef0c557a3ae79a) 반영해두었어!